### PR TITLE
Update "feature request" GitHub issue template

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ about: Suggest an idea for this project
 
 ### Is your feature request related to a problem? Please describe.
 A clear and concise description of what the problem is.
-Example. I'm am trying to do [...] but [...]
+Example: I am trying to do [...] but [...]
 
 ### Describe the solution you'd like
 A clear and concise description of what you want to happen.
```
Comment (loic-sharma) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Personally I'd fold the `Describe alternatives you've considered` section into this one. Something like:

```suggestion
 A clear and concise description of what you want to happen. Include any alternative solutions you've considered.
```

Comment (jkotalik) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Go for it.

**Review by jkotalik — APPROVED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (loic-sharma) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Done!

**Comment by loic-sharma** [comment]
@jkotalik I've updated the pull request

**Pull request merged**